src/controllers/ranking.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from "express";
+import { AuthenticatedRequestParams } from "../interfaces/authenticatedRequestParams";
+import { UserModel } from "../models/user.model";
+
+export const getRanking = async (  _req: Request<AuthenticatedRequestParams>,
+    res: Response) => {
+    try {
+        const users = await UserModel.find().select('derbyName earnedPoints').sort({ earnedPoints: 1 });
+        res.status(200).json({ isFinal: true, ranking: users});
```

Comment:
isFinal should be dependant on whether all games already have been completed/points set for all games, rather than being a static true (unless you did it in another PR sorry, I'm reviewing in order 😅 )

Reply:
Thanks, I had forgotten it 😅 Done